src/pages/CreateShelter/CreateShelter.tsx

```diff
@@ -107,21 +108,11 @@ const CreateShelterComponent = () => {
   }, [cepData, setFieldValue, setErrors]);
 
   return (
-    <div className="flex flex-col h-screen items-center">
-      <Header
-        title="Cadastrar novo abrigo"
-        className="bg-white [&_*]:text-zinc-800 border-b-[1px] border-b-border"
-        startAdornment={
-          <Button
-            variant="ghost"
-            className="[&_svg]:stroke-blue-500"
-            onClick={() => navigate('/')}
-          >
-            <ChevronLeft size={20} />
-          </Button>
-        }
-      />
-      <div className="p-4 flex flex-col max-w-5xl w-full gap-3 items-start h-full">
+    <SecondaryLayout
```

Comment:
Qual a diferença do `MainLayout` e do `SecondaryLayout`?

Reply:
O `MainLayout` seria o layout principal da app, exemplo a home page. Ja o `SecondaryLayout` se voltado mais para paginas internas. Talvez os nomes não estejam tao bons, mas podemos pensar em algo melhor.

Reply:
Se for para páginas internas talvez um nome que remeta a isso possa ajudar. Algo como `InnerPagesLayout` ou `InternalPagesLayout`.
O que acha?